examples/refresh.py

```diff
@@ -0,0 +1,126 @@
+import marimo
+
+__generated_with = "0.1.68"
+app = marimo.App(width="full")
+
+
+@app.cell
+def __(
+    chart,
+    iss_df,
+    mo,
+    n_points_slider,
+    refresh_interval_slider,
+    refresher,
+):
+    mo.hstack([
+        mo.vstack([
+            mo.md("## Settings | [`marimo.ui.slider`](https://docs.marimo.io/api/inputs/slider.html), [`marimo.ui.refresh`](https://docs.marimo.io/recipes.html#run-a-cell-on-a-timer)\n---"),
+            refresh_interval_slider,
+            n_points_slider,
+            refresher,
+            mo.md("## ISS Positions | [`marimo.ui.altair_chart`](https://docs.marimo.io/api/plotting.html#marimo.ui.altair_chart)\n---"),
+            mo.as_html(chart)
```

Comment:
The width not being respected is a bug on our part I think. Oops! In the meantime, this suggestion should fix it:

```suggestion
            mo.as_html(chart).style({"width": "700px"})
```